mezzanine/compat.py

```diff
@@ -0,0 +1,32 @@
+import django
+
+
+_django_ver = django.VERSION
+
+is_dj2 = (_django_ver[0] == 2)
+
+if is_dj2:
```

Comment:
Normally in Python is [easier to ask for forgiveness then ask for permission](https://docs.python.org/3.6/glossary.html#term-eafp). For that reason here is better to use `try:..except:` instead of checking if this is Django 2 or not.